azext_iot/central/_help.py

```diff
@@ -1301,7 +1542,45 @@ def _load_central_monitors_help():
         "iot central device twin show"
     ] = """
         type: command
-        short-summary: Get the device twin from IoT Hub.
+        short-summary: Get the device properties from IoT Central.
+    """
+
+    helps[
+        "iot central device twin update"
+    ] = """
+        type: command
+        short-summary: Manage the device properties from IoT Central.
+        examples:
+        - name: Update device properties
+          text: >
+            az iot central device twin update
+            --app-id {appid}
+            --device-id {deviceid}
+            -k {content}
+        - name: Replace device properties
+          text: >
+            az iot central device twin update
+            --app-id {appid}
+            --device-id {deviceid}
+            -r {replace}
```

Comment:
can have the replace be a flag (as in just have -r without the {replace}

Reply:
Let me try